cypress/support/commands.ts

```diff
@@ -183,6 +183,28 @@ Cypress.Commands.add('keyboardControls', (triggerKeys: any = {}, count = 1) => {
   }
 });
 
+/**
+ * Intercept all requests and return
```

Comment:
In case of multiple intercepted requests is the wrapper going to be overlapping if using the same term?
An alternative may be to parse and chain the method and URL in a camel case with `@intercept` as base?